src/MiScript/Ast/Expressions/CallExpression.cs

```diff
@@ -1,12 +1,13 @@
 ﻿using System.Collections.Generic;
 using System.Linq;
+using Miki.Localization;
 using MiScript.Models;
 
 namespace MiScript.Ast
 {
-    public class CallStatement : Statement
+    public class CallExpression : Expression
```

Comment:
What's the reason for the change?

Reply:
Because function calls can now return a value (e.g. `say upper("bar")`), so they are expressions now.